src/Api/Basic/Initializer.php

```diff
@@ -101,6 +101,12 @@ static public function setUpRoutes(array $opts = []): string {
             throw new \RuntimeException(sprintf('Request object not initialized. \%s', __METHOD__));
         }
 
+        try {
+            $config = \Maleficarum\Ioc\Container::getDependency('Maleficarum\Config');
+        } catch (\RuntimeException $e) {
+            throw new \RuntimeException(sprintf('Config object not initialized. \%s', __METHOD__));
```

Comment:
If you decide to re-throw it perhaps it would be worth to set the `$previous` param when creating new exception.